applications/GeoMechanicsApplication/tests/test_helper.py

```diff
@@ -277,6 +277,23 @@ def get_cauchy_stress_tensor(simulation):
 
     return cauchy_stress_tensors
 
+
+def get_cauchy_stress_vectors(simulation):
+    model_part = simulation._list_of_output_processes[0].model_part
```

Comment:
Can we use something like the main modelpart or PorousDomain or ...
to be independent of the output process?

Reply:
I now removed these new functions and used the get_on_integration_points that was already there. It has the same issue, but since it could change the behavior of tests, I propose to do that at another time